client/login/wp-login/index.jsx

```diff
@@ -89,7 +103,7 @@ class Login extends React.Component {
 		}
 	};
 
-	footerLinks() {
+	footerLinks = () => {
```

Comment:
As Yury mentioned below, you should use enzyme to test React components and treat them more like a black box. You can always extract footer links to its own component if you want to test that part explicitly.